yoti-sdk-api/src/main/java/com/yoti/json/ResourceMapper.java

```diff
@@ -0,0 +1,36 @@
+package com.yoti.json;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.annotation.Nulls;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.cfg.MapperConfig;
+import com.fasterxml.jackson.databind.introspect.VisibilityChecker;
+
+public class ResourceMapper {
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    static {
+        MAPPER.disable(DeserializationFeature.WRAP_EXCEPTIONS)
+                .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+                .disable(DeserializationFeature.READ_DATE_TIMESTAMPS_AS_NANOSECONDS)
+                .setVisibility(configureVisibility(MAPPER.getDeserializationConfig()))
+                .setDefaultSetterInfo(JsonSetter.Value.forValueNulls(Nulls.SKIP));
+    }
+
+    private static VisibilityChecker<?> configureVisibility(MapperConfig<?> config) {
+        return config.getDefaultVisibilityChecker()
+                .withFieldVisibility(JsonAutoDetect.Visibility.NONE)
+                .withGetterVisibility(JsonAutoDetect.Visibility.NONE)
+                .withIsGetterVisibility(JsonAutoDetect.Visibility.NONE)
+                .withSetterVisibility(JsonAutoDetect.Visibility.NONE)
+                .withCreatorVisibility(JsonAutoDetect.Visibility.NONE);
+    }
+
+    public static ObjectMapper mapper() {
```

Comment:
```java
ResourceMapper.mapper()
```
stutters... maybe `getInstance()` or simply move the `writeValueAsString` (and any other `ObjectMapper` methods used) here and make them static methods so one does `ResourceMapper.doSomething(,,,)`